superchain/configs/sepolia/mode.toml

```diff
@@ -7,7 +7,7 @@ superchain_level = 0
 standard_chain_candidate = true # This is a temporary field which causes most of the standard validation checks to run on this chain
 superchain_time = 1703203200 # Fri 22 Dec 2023 00:00:00 UTC
 batch_inbox_addr = "0xcDDaE6148dA1E003C230E4527f9baEdc8a204e7E"
-canyon_time = 1703203200 # Fri 22 Dec 2023 00:00:00 UTC
+canyon_time = 1703203204 # Fri 22 Dec 2023 00:00:00 UTC
```

Comment:
We're running mode Sepolia chains with the old timestamp. Why would you want to change this?

Note also that the comment is the old time, not the +4s to what you change it here.